docs/PFCWD-test-plan.md

```diff
@@ -0,0 +1,189 @@
+
+### PFC Polling Interval 
+
+PFC Polling interval is the the frequency at which queues should be checked for congestion or recovery detection. By default, polling interval is calculated internally or it considers the value configured through CLI.
+
+### 1.1.1 Scope
+
+The test cases depicted in this document aim to do functional testing of PFC watchdog behavior of SONiC DUT (Device Under Test).
+
+### 1.1.2 Testbed
+#### 1.1.2.1 Topology 1
+```
++-----------------+           +--------------+           +-----------------+       
+| Keysight Port 1 |------ et1 |   SONiC DUT  | et2 ------| Keysight Port 2 | 
++-----------------+           +--------------+           +-----------------+ 
+                                   et3
+                                    |
+                                    |
+                                    |
+                            +-----------------+
+                            | Keysight Port 3 |
+                            +-----------------+
+```
+*Fig. 1: Topology 1*
+
+## 1.2 Setup configuration
+
+### 1.2.1 DUT Configuration
+•PFC watchdog and PFC must be enabled in the DUT by default.
+
+
+### 1.2.2 Keysight configuration
+•All Keysight ports should have the same bandwidth capacity.
+
+•	Test specific configurations are mentioned in respective test cases.
+
+## 1.3 Test Cases
+
+### 1.3.1 Test Case #1 - PFCWD two senders two receivers
+
+#### 1.3.1.1 Test Objective
+
+This test aims to verify how PFC watchdog can handle PFC storms in a topology with two senders and two receivers.
+
+#### 1.3.1.2 Test Topology
+
+Refer to Topology 1.1.2.1 for the test topology.
+
+#### 1.3.1.3 Test Configuration
+
+- On SONiC DUT configure the following:
+  1. Enable watchdog with default storm detection time (400ms) and restoration time (2sec). Please note that different switches have different detection times and restoration times, e.g., 200ms and 400ms.  
+  2. Configure a single lossless priority value Pi (0 <= i <= 7).
+  3. To minimize configuration complexity, it is recommended that the SONiC DUT be configured as either Top of Rack (ToR) / Tier 0 (T0) switch with three VLAN interfaces.
+
+- Configure following traffic items on the Keysight device:
+  1. Traffic 1<->2 : Bi-directional traffic between Keysight port 1 and port 2, with DSCP value mapped to lossless priority Pi configured in the DUT. Traffic Tx rate should be configured as 50% of line rate.
+  2. Traffic 2<->3 : Bi-directional traffic between Keysight port 2 and port 3, with DSCP value mapped to lossless priority Pi configured in the DUT. Traffic Tx rate should be configured as 50% of line rate.
+  3. PFC pause storm: Persistent PFC pause frames from Keysight
+        port 3 to et3 of DUT. Priority of the PFC pause
+        frames should be same as that configured in DUT and the
```

Comment:
same as the lossless priority configured at DUT

Reply:
taken care of